platforms/ios/src/TangramMap/platform_ios.mm

```diff
@@ -115,17 +115,36 @@ void initGLExtensions() {
     return data;
 }
 
+bool allowedFamily(NSString* familyName) {
+    const NSArray<NSString *> *allowedFamilyList = @[ @"Hebrew", @"Kohinoor", @"Gumurki", @"Thonburi", @"Tamil",
+                                                    @"Gurmukhi", @"Kailasa", @"Sangam", @"PingFang", @"Geeza",
+                                                    @"Mishafi", @"Farah", @"Hiragino" ];
+
+    for (NSString* allowedFamily in allowedFamilyList) {
+        if ( [familyName containsString:allowedFamily] ) { return true; }
+    }
+    return false;
+}
+
 std::vector<FontSourceHandle> iOSPlatform::systemFontFallbacksHandle() const {
-    NSArray* fallbacks = [UIFont familyNames];
+
+    NSArray<NSString *> *fallbacks = [UIFont familyNames];
 
     std::vector<FontSourceHandle> handles;
 
     for (id fallback in fallbacks) {
-
-        handles.emplace_back([fallback]() {
-            auto data = loadUIFont([UIFont fontWithName:fallback size:1.0]);
-            return data;
-        });
+    
+        if (!allowedFamily(fallback)) { continue; }
+
+        for (id fontName in [UIFont fontNamesForFamilyName:fallback]) {
```

Comment:
I generally try *really* hard to avoid using `id` for things if possible.

Looking at https://developer.apple.com/reference/uikit/uifont/1619023-fontnamesforfamilyname it's returning an array of NSStrings, so I'd rather see that type explicitly set if possible.